templates/modules/footer.html

```diff
@@ -37,6 +37,24 @@
       th:href="${theme.config.beian.icp_link}"
       th:text="${theme.config.beian.icp_text}"
     ></a>
+  <span class="container mx-auto block text-center text-sm text-gray-500 dark:text-slate-300">
+    <style>
+      .text-with-image {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        gap: 10px;
+      }
+    </style>
+    <div class="center">
+      <p class="text-with-image">
+        <img th:if="${not #strings.isEmpty(theme.config.beian.gongan_text)}"
+        src="/themes/theme-earth/assets/images/gongan_beian.png">
```

Comment:
这里写死可能不太好，我们在内部封装了 assets 的调用，可以参考：https://docs.halo.run/developer-guide/theme/static-resources

```suggestion
        th:src="@{/assets/images/gongan_beian.png}">
```